packages/paste-website/src/pages/components/badge/index.mdx

```diff
@@ -335,6 +357,34 @@ of the Popover docs to add a Popover to a Badge.
   {badgeModalExample}
 </LivePreview>
 
+### Small Badges
+
+Pass `size="small"` if you need a slightly smaller-sized Badge. Use the Small Badge sparingly as it's reserved for specific use cases only.
+
+<LivePreview scope={{Box, Badge, InformationIcon, ErrorIcon, WarningIcon, SuccessIcon, NewIcon}} language="jsx">
+  {`<Box display="flex" columnGap="space40" rowGap="space60" flexWrap="wrap">
+  <Badge as="span" size="small" variant="neutral">
+    Neutral
+  </Badge>
+  <Badge as="span" size="small" variant="warning">
+    Warning
+  </Badge>
+  <Badge as="span" size="small" variant="error">
+    Error
+  </Badge>
+  <Badge as="span" size="small" variant="success">
+    Success
+  </Badge>
+  <Badge as="span" size="small" variant="new">
+    New
+  </Badge>
+  <Badge as="span" size="small" variant="subaccount">
+    Subaccount
+  </Badge>
+</Box>
+`.trim()}
+</LivePreview>
+
 ---
 
```

Comment:
(can't make a comment on the props table so leaving this here)

todo: update props table

Reply:
ah yes ty